shielddash/core/static/js/app/api/study-api.js

```diff
@@ -5,18 +5,17 @@ import {
   getStudyFetching, getStudySuccess, getStudyFailure
 } from '../actions/study-actions';
 
-const mockEndpoints = {
-  GET_STUDIES: 'http://localhost:3009/studies',
-  GET_STUDY: 'http://localhost:3009/studys/'
+const devEndpoints = {
+  GET_STUDIES: 'http://localhost:8000/studies/',
+  GET_STUDY: 'http://localhost:8000/studies/'
 };
 
 const prodEndpoints = {
   GET_STUDIES: 'https://shielddash.herokuapp.com/studies/',
   GET_STUDY: 'https://shielddash.herokuapp.com/studies/'
 }
 
-// TODO: Check node environment to set this.
-const endpoints = process.env.NODE_ENV === 'dev' ? mockEndpoints : prodEndpoints;
+const endpoints = __PROD_API__ ? prodEndpoints : devEndpoints;
```

Comment:
Gotcha. That makes sense. And I think it makes sense to default to the dev server. Works for me!